source/postprocess/gravity_point_values.cc

```diff
@@ -495,50 +524,50 @@ namespace aspect
                              "solution from noise due to the model grid.");
           prm.declare_entry ("Number points radius", "1",
                              Patterns::Integer (0),
-                             "Parameter for the uniform distribution sampling scheme: "
+                             "Parameter for the map sampling scheme: "
                              "This specifies the number of points along "
                              "the radius (e.g. depth profile) between a minimum and "
                              "maximum radius.");
           prm.declare_entry ("Number points longitude", "1",
                              Patterns::Integer (0),
-                             "Parameter for the uniform distribution sampling scheme: "
+                             "Parameter for the map sampling scheme: "
                              "This specifies the number of points along "
                              "the longitude (e.g. gravity map) between a minimum and "
                              "maximum longitude.");
           prm.declare_entry ("Number points latitude", "1",
                              Patterns::Integer (0),
-                             "Parameter for the uniform distribution sampling scheme: "
+                             "Parameter for the map sampling scheme: "
                              "This specifies the number of points along "
                              "the latitude (e.g. gravity map) between a minimum and "
                              "maximum latitude.");
           prm.declare_entry ("Minimum radius", "0.",
                              Patterns::Double (0.0),
-                             "Parameter for the uniform distribution sampling scheme: "
+                             "Parameter for the map sampling scheme: "
                              "Minimum radius may be defined in or outside the model. "
                              "Prescribe a minimum radius for a sampling coverage at a "
                              "specific height.");
           prm.declare_entry ("Maximum radius", "0.",
                              Patterns::Double (0.0),
-                             "Parameter for the uniform distribution sampling scheme: "
+                             "Parameter for the map sampling scheme: "
                              "Maximum radius can be defined in or outside the model.");
-          prm.declare_entry ("Minimum longitude", "-180.",
-                             Patterns::Double (-180.0, 180.0),
-                             "Parameter for the uniform distribution sampling scheme: "
+          prm.declare_entry ("Minimum longitude", "-180",
+                             Patterns::Double (-180.0,180.0),
```

Comment:
Keep these two lines as they were before (with dots and spaces). Same for the parameters below.